rgw/admin/user_test.go

```diff
@@ -135,3 +138,24 @@ func (suite *RadosGWTestSuite) TestUser() {
 		assert.NoError(suite.T(), err)
 	})
 }
+
+func TestGetUserMockAPI(t *testing.T) {
+	r := ioutil.NopCloser(bytes.NewReader(fakeUserResponse))
+	mockClient := &MockClient{
+		MockDo: func(req *http.Request) (*http.Response, error) {
+			if req.URL.RawQuery == "format=json&uid=dashboard-admin" && req.Method == "GET" && req.URL.Path == "127.0.0.1/admin/user" {
```

Comment:
Possibly a holdover from older code but should "GET" be swapped to a constant from the http package?

Reply:
This was introduced with the new interface and the mocking. I can change that.